src/js/controller/instream-html5.js

```diff
@@ -145,6 +145,10 @@ define([
                 provider.mute(_model.get('mute'));
 
                 _adModel.on('change:state', changeStateEvent, _this);
+            } else if(provider.state === 'playing') {
```

Comment:
@egreaves @robwalch - what do you think of this

```js
if (_isMobile && _isFirstPlay) {
    // do nothing, since we need to wait for a touch event
} else {
     _this.setState(states.LOADING);
}
```